components/camel-sjms/src/test/java/org/apache/camel/component/sjms/producer/QueueProducerQoSTest.java

```diff
@@ -106,8 +99,9 @@ public void testInOutQueueProducerTTL() throws Exception {
 
         MockEndpoint.assertIsSatisfied(context);
 
-        DestinationViewMBean queue = service.getQueueMBean(TEST_INOUT_DESTINATION_NAME);
-        assertEquals(0, queue.getQueueSize(),
+        //TODO - I've exposed service.getMBeanServer(), do you think it  can be used here?
```

Comment:
Maybe a leftover here?

Reply:
Exactly, I tried to re-implement that test with JMX but Artemis expose a ManagementService that did the trick.